climt/_components/surface_ice.py

```diff
@@ -296,18 +305,24 @@ def calculate_new_ice_temperature(self, rho, specific_heat, kappa,
         a_sub = np.zeros(num_layers)
         a_sup = np.zeros(num_layers)
 
-        K_bar = 0.25*(kappa[2:] + kappa[:-2]) + 0.5 * kappa[1:-1]
-        K_mid = 0.5*(kappa[1:] + kappa[:-1])
+        # K_bar = 0.25*(kappa[2:] + kappa[:-2]) + 0.5 * kappa[1:-1]
+        # K_mid = 0.5*(kappa[1:] + kappa[:-1])
+        K_int = 0.5*(kappa[:-1] + kappa[1:])
```

Comment:
I would suggest re-naming `_int` to `_interface` wherever it appears. When I first read this I was wondering if this had to do with it being cast to an integer, and it doesn't appear this change would bloat any lines.